src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellPageRendererTracker.cs

```diff
@@ -22,15 +19,25 @@ public class ShellPageRendererTracker : IShellPageRendererTracker, IFlyoutBehavi
 
 		public bool IsRootPage { get; set; }
 
+#nullable disable
```

Comment:
why are adding this  here instead of fixing the all file?

Reply:
Because this properties are implementing the `IShellPageRendererTracker` interface. To add nullability here, we need to update the interface too.